pkg/operator/encryptionprovider/encyrptionprovider.go

```diff
@@ -40,11 +40,11 @@ func New(
 // EncryptedGRs returns resources that need to be encrypted
 // Note: the list can change depending on the existence and attached annotations of encryption-config-openshift-oauth-apiserver in openshift-config-managed namespace as described in https://github.com/openshift/enhancements/blob/master/enhancements/etcd/etcd-encryption-for-separate-oauth-apis.md
 //
-// case 1 encryption off or the secret was annotated - return an empty list of EncryptedGRs
-// case 2 otherwise return the authoritative list of EncryptedGRs
+// case 1 encryption off or the secret wasn't annotated - return the authoritative list of EncryptedGRs
+// case 2 otherwise return an empty list of EncryptedGRs - OAS-O is still in charge
```

Comment:
please be explicit here: what does "otherwise" actually mean?

Reply:
updated.